client/src/components/utils/Navbar/index.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import logoImageLight from 'assets/FNLogoLightV2.png';
+import logoImageDark from 'assets/FNLogodarkV2.png';
+import './style.css';
```

Comment:
Have you thougght about using CSS modules?

I think it should work by default with create-react-app by just renaming `style.css` to `style.module.css`. Then you can:

```js
import styles from 'style.module.css'
```

and later for example:

```
<div className="{styles.navbar__menu}">
```

The benefit of this approach is that it will allow you to avoid CSS variable conflicts without manually having to include a prefix. For example, with the example above, you could simply name the class `menu` instead of `navbar__menu`.

cc @ahmedisam99

Reply:
Does this method implicitly add a prefix to the class name?

Reply:
@ahmedisam99 Yep, it adds a prefix (e.g. `navbar` if the file is named `navbar.module.css`) and also a hash to avoid conflicts.

Reply:
nice, using this it would be much easier writing and refactoring CSS actually!

thx @poiru :+1: